examples/experimental/dagster-airlift/examples/dbt-example/dbt_example/dagster_defs/observe.py

```diff
@@ -40,15 +42,23 @@
 defs = build_defs_from_airflow_instance(
     airflow_instance=airflow_instance,
     defs=combine_defs(
-        *specs_from_lakehouse(
-            dag_id="load_lakehouse",
-            task_id="load_iris",
-            csv_path=CSV_PATH,
+        dag_defs(
+            "load_lakehouse",
+            task_defs(
+                "load_iris",
+                *specs_from_lakehouse(
```

Comment:
potentially some weirdness about the need to use the splat operator in some circumstances and not others, but I guess the fact that this function is part of the example, which should make it clear.

I do worry that users will start getting a bit confused about Definitions objects flying around in addition to assets/checks. I wonder if it would be more clear to actually just "bind" the resources to the assets themselves, and provide a flat list of definitions, and have the Definitions merging be allowed but not first class.

I guess alternative would be to have a flat list of defs that you can provide, and then additional Definitions objects.
Also possible that I'm just overcomplicating it.